src/app/public/modules/repeater/repeater.service.ts

```diff
@@ -3,15 +3,69 @@ import {
   Injectable
 } from '@angular/core';
 
+import {
+  BehaviorSubject
+} from 'rxjs/BehaviorSubject';
+
 import {
   SkyRepeaterItemComponent
 } from './repeater-item.component';
 
 @Injectable()
 export class SkyRepeaterService {
+
+  public activeItemId: BehaviorSubject<string> = new BehaviorSubject(undefined);
+
   public itemCollapseStateChange = new EventEmitter<SkyRepeaterItemComponent>();
 
-  public onItemCollapseStateChange(item: SkyRepeaterItemComponent) {
+  public items: BehaviorSubject<Array<SkyRepeaterItemComponent>> = new BehaviorSubject<Array<SkyRepeaterItemComponent>>([]);
```

Comment:
I know you probably duplicated the same pattern from tabs, but is there another reason why we need to make the `items` property a `BehaviorSubject`? In my opinion, it adds more complexity than it's worth.

Reply:
I was a little hesitant on this, due to a lot of functionality being asynchronous. However, I removed it and it appears to be working fine without the `BehaviorSubject`. Unit tests and manual tests still work great. Think it's ok?